index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Md. Porag Sarkar: Portfolio</title>
+</head>
+
+<body>
+    <div class="container-parent">
+
+        <header class="top-navigation">
+            <div class="container-logo">
+                <p class="logo">My Logo</p>
+            </div>
+            <img class="menu-icon" src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
```

Comment:
- [ ] Your hamburger button looks nice and is at the correct position :100:  However let's wrap it with a `nav` tag for semantics reasons and the right position for convention and user experience for mobile devices.